scripts/rollup.config.js

```diff
@@ -7,45 +7,32 @@ const config = generate(options);
 
 // Add additonal builds/customization here!
 
-config.builds.pad = config.makeBuild('browser', {
-  input: 'src/pad.js',
-  output: [{
-    name: 'pkcs7.pad',
-    file: 'dist/pkcs7.pad.js',
-    format: 'umd'
-  }]
-});
-
-config.builds.unpad = config.makeBuild('browser', {
-  input: 'src/unpad.js',
-  output: [{
-    name: 'pkcs7.unpad',
-    file: 'dist/pkcs7.unpad.js',
-    format: 'umd'
-  }]
-});
-
-if (config.builds.browserMin) {
-  config.builds.padMin = config.makeBuild('browser', {
+config.builds = Object.assign(config.builds, {
+  pad: config.makeBuild('browser', {
     input: 'src/pad.js',
     output: [{
+      name: 'pkcs7.pad',
+      file: 'dist/pkcs7.pad.js',
+      format: 'umd'
+    }, {
       name: 'pkcs7.pad',
       file: 'dist/pkcs7.pad.min.js',
       format: 'umd'
-    }],
-    plugins: config.builds.browserMin.plugins
-  });
-
-  config.builds.unpadMin = config.makeBuild('browser', {
+    }]
+  }),
+  unpad: config.makeBuild('browser', {
     input: 'src/unpad.js',
     output: [{
+      name: 'pkcs7.unpad',
```

Comment:
we minify all files named `.min.js` now. so we can make the build much faster by using two outputs.